### PR TITLE
Fixed layout edit button and call_required_libraries implementation

### PR DESCRIPTION
**Pull request by Trodrige**
Fix for #668 and #681.

### CODE REVIEW

File: interface/super/edit_layout.php [hunk 1]
```diff
@@ -958,7 +958,7 @@ function cidChanged(lino, seq) {
   if ($row['group_name'] != $prevgroup) {
     if ($firstgroup == false) { echo "</tbody></table></div>\n"; }
     echo "<div id='".$row['group_name']."' class='group'>";
```
Comment (aethelwulffe) [inline interface/super/edit_layout.php]: Need to test this a variable resolutions.

File: sites/default/sqlconf.php [hunk 2]
```diff
@@ -1,31 +0,0 @@
-<?php
```
Comment (aethelwulffe) [inline sites/default/sqlconf.php]: Dude...can't delete the sqlconf.php file!  (not yet anyway!)  Should have been a gitignore, if you are running your test instance from your repo.
  If you like, you could make a mechanism that generates this file at setup...but you got to put this back before we can merge it.

**Comment by aethelwulffe** [comment]
One other note:  Try to keep different feature changes in different branches if you can.  It is very tempting to sneak in a little thing here or there...just try to resist.

**Comment by Trodrige** [comment]
@aethelwulffe I've re-included sqlconf.php.

**Comment by teryhill** [comment]
Testing well

image

**Comment by aethelwulffe** [comment]
OK @Trodrige  Looks like you have your first clean commit.  Merging.

**Pull request merged**